src/benchmarks/gc/src/suite.py

```diff
@@ -215,22 +215,46 @@ def _normal_benchmarks(proc_count: int) -> Mapping[str, Benchmark]:
     tagb_factor = 0.5 if proc_count == 1 else 1
     return {
         "0gb": Benchmark(
-            arguments=GCPerfSimArgs(tc=proc_count, tagb=300 * tagb_factor, tlgb=0), min_seconds=10
+            arguments=GCPerfSimArgs(
+                tc=proc_count,
+                tagb=300 * tagb_factor,
+                tlgb=0
+            ),
+            min_seconds=10
         ),
         "2gb": Benchmark(
-            arguments=GCPerfSimArgs(tc=proc_count, tagb=300 * tagb_factor, tlgb=2, sohsi=50)
+            arguments=GCPerfSimArgs(
+                tc=proc_count,
+                tagb=300 * tagb_factor,
+                tlgb=2,
+                sohsi=50,
+                pohsi=5,
```

Comment:
do we have to specify poh args here? if we don't, doesn't that just mean there's nothing on the poh?

Reply:
This is an interesting one. For all flags not passed, GCPerfSim sets a default value. Due to Python class semantics, since I had to add them for the 0GB benchmarks, it was much simpler to add them to all benchmarks. I checked these default values with the debugger beforehand.